app/api/events.py

```diff
@@ -380,8 +380,8 @@ def before_get_object(self, view_kwargs):
 class EventRelationship(ResourceRelationship):
 
     def before_get_object(self, view_kwargs):
-        if view_kwargs.get('identifier'):
-            event = self.session.query(Event).filter_by(identifier=view_kwargs['identifier']).one()
```

Comment:
@enigmaeth, can you use safe_query here, looks like I forgot it here :sweat_smile: